New ui

jcesarmobile opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
New UI with a new flash button with icons that alternate when pressing them and change the flash mode (if available) and new switch camera button to change from front to rear camera.

## Related Issue
<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?

## Screenshots (if appropriate):
![img_0057](https://user-images.githubusercontent.com/1637892/33141331-21d38366-cfb3-11e7-93e9-87ea86a78902.PNG)



## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

jcesarmobile committed:
Add new icons

jcesarmobile committed:
Added toolbar with cancel and flash buttons, changed cancel to switch camera

jcesarmobile committed:
Switch camera and change flash features

jcesarmobile committed:
Only set flash mode and show buttons if supported

jcesarmobile committed:
Adjust flash button size to be closer to the right

macdonst reviewed (changes requested):
@jcesarmobile can you make the switch camera button bigger or make the touch target bigger? I was having problems if I didn't click exactly on the icon. The iOS camera app is using a bigger camera icon.

As for the cancel button, since it is in such a small space it can't rotate so maybe we should use an X instead?

jcesarmobile commented:
As changes in xib are not persisted with --link, I think I forgot to change the switch camera width. Changed from 16 to 44, should be easier to tap now.
Changed the Cancel to an ╳ icon, if you think it's too thin, maybe I can use an image instead, but didn't like the X letter

jcesarmobile committed:
Make switch camera bigger and change cancel with x icon
